KubeArmor/core/kubeArmor.go

```diff
@@ -339,39 +352,33 @@ func KubeArmor() {
 		// wait for a while
 		time.Sleep(time.Second * 1)
 
-		for dm.Node.NodeIP == "" {
-			kg.Print("The node information is not updated yet")
-
-			// wait for a while
-			time.Sleep(time.Second * 1)
-		}
-
-		dm.Node.EnableKubeArmorPolicy = cfg.GlobalCfg.Policy
-		dm.Node.EnableKubeArmorHostPolicy = cfg.GlobalCfg.HostPolicy
-	} else {
-		dm.Node.NodeName = kl.GetHostName()
-		dm.Node.NodeIP = kl.GetExternalIPAddr()
+		if dm.Node.NodeIP == "" {
+			for timeout := 0; timeout <= 60; timeout++ {
+				if dm.Node.NodeIP != "" {
+					break
+				}
 
-		dm.Node.KernelVersion = kl.GetCommandOutputWithoutErr("uname", []string{"-r"})
-		dm.Node.KernelVersion = strings.TrimSuffix(dm.Node.KernelVersion, "\n")
+				if dm.Node.NodeIP == "" && timeout == 60 {
+					kg.Print("The node information is not available")
```

Comment:
It is better to print this info every second ... This info usually helps me in knowing that `kubectl proxy &` is not executed.

Reply:
@nyrahul Added the update as you commented.

Reply:
@nyrahul Added the update as you commented.